nucypher/cli/commands/worklock.py

```diff
@@ -78,101 +95,123 @@ def worklock():
+
+    if not worklock_options.bidder_address:
+        worklock_options.bidder_address = select_client_account(emitter=emitter,
+                                                                provider_uri=registry_options.provider_uri,
+                                                                network=registry_options.network,
+                                                                show_balances=True)
     if not value:
         if force:
             raise click.MissingParameter("Missing --value.")
-        value = int(Web3.fromWei(click.prompt("Enter bid amount in ETH", type=click.FloatRange(min=0)), 'wei'))
+        value = click.prompt("Enter bid amount in ETH", type=click.STRING)
+    value = int(Web3.toWei(Decimal(value), 'ether'))
 
-    if not worklock_options.bidder_address:
-        worklock_options.bidder_address = select_client_account(emitter=emitter,
-                                                                provider_uri=general_config.provider_uri)
     registry = registry_options.get_registry(emitter, general_config.debug)
-    bidder = worklock_options.create_bidder(registry=registry)
+    bidder = worklock_options.create_bidder(registry=registry, hw_wallet=hw_wallet)
 
     if not force:
         paint_bidding_notice(emitter=emitter, bidder=bidder)
-        click.confirm(f"Place WorkLock bid of {Web3.fromWei(value, 'ether')} ETH?", abort=True)
+        click.confirm(f"Place WorkLock bid of {prettify_eth_amount(value)}?", abort=True)
 
     receipt = bidder.place_bid(value=value)
     emitter.message("Publishing WorkLock Bid...")
 
     # Ensure the total bid value is worth a claim that is at
     # least large enough for the minimum stake.
     minimum = bidder.economics.minimum_allowed_locked
-    available_claim = bidder.available_claim
+    available_claim = NU.from_nunits(bidder.available_claim)
     if available_claim < minimum:
-        warning = f"Total bid is too small for a claim, please bid more or cancel. " \
-                  f"{available_claim} total / {minimum} minimum" \
-                  f"(Total must be worth at least {NU.from_nunits(minimum)})"
+        warning = f"Total bid ({available_claim}) is too small for a claim, please bid more or cancel.\n" \
+                  f"Total must be worth at least {NU.from_nunits(minimum)})."
         emitter.echo(warning, color='yellow')
     else:
-        message = f'Current bid: {bidder.get_deposited_eth} | ' \
-                  f'Available Claim: {bidder.available_claim} |' \
-                  f'Note that available claim value may fluctuate ' \
-                  f'until bidding closes and claims are finalized.'
+        message = f'Current bid: {prettify_eth_amount(bidder.get_deposited_eth)} | ' \
+                  f'Available claim: {available_claim}\n' \
+                  f'Note that available claim value may fluctuate until bidding closes and claims are finalized.\n'
         emitter.echo(message, color='yellow')
```

Comment:
Perhaps we should use a different colour than yellow for the successful bid, since the warning (claim < minimum) in the `if` block above is already yellow. 

The bid/claim values can be in green(? or something else), and the `note` in a different colour (blue?) to make it pop...?